docs/src/app/components/pages/discover-more/Showcase.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import GridList from 'material-ui/lib/grid-list/grid-list';
+import GridTile from 'material-ui/lib/grid-list/grid-tile';
+
+import MarkdownElement from '../../MarkdownElement';
+import showcaseText from './showcase.md';
+
+const styles = {
+  gridList: {
+    margin: 10,
+  },
+  gridTile: {
+    cursor: 'pointer',
+  },
+};
+
+const appList = [
+  // Under development
+  // {
+  //   title: 'Call-Em-All',
+  //   author: 'Call-Em-All',
+  //   img: 'images/showcase/callemall.png',
+  //   link: '',
+  // },
+  {
+    title: 'SplitMe',
+    author: 'Olivier Tassinari',
+    img: 'images/showcase/splitme.png',
+    link: 'https://splitme.net/',
```

Comment:
For now, I just have an Android app. The web version is not production ready yet. I miss the service worker logic to work offline. Still, you can have a look https://splitme.net/#/accounts.